cogni-libs/src/test/java/zone/cogni/libs/core/utils/DateFormatsTest.java

```diff
@@ -55,7 +55,7 @@ public void parse_xsd_date_only_seconds() {
   public void fixed_date_as_xsd_datetime_format() {
     // given
     LocalDateTime localDateTime = LocalDateTime.of(2015, Month.AUGUST, 20, 6, 30);
-    ZoneId cet = ZoneId.systemDefault();
+    ZoneId cet = ZoneId.of("Europe/Paris");
```

Comment:
This is because I changed it directly on develop branch in the hope it would fix the problem. Was not a good idea, I admit :)